bats/fb-content-katello.bats

```diff
@@ -82,7 +77,7 @@ setup() {
 @test "create activation key" {
   hammer -u admin -p changeme activation-key create --organization="Default Organization" \
     --name="Test AK" --content-view="Test CV" --lifecycle-environment="Test" \
-    --unlimited-hosts | grep -q "Activation key created"
+    --unlimited-content-hosts | grep -q "Activation key created"
```

Comment:
> Have you tested this against nightly bats? I'd think this would break our nightly testing given you are changing a parameter here.

I have not, the parameter did not actually work in Sat 6.2 beta...

-- 
Later,
 Lukas #lzap Zapletal